qermit/noise_model/transpiler_backend.py

```diff
@@ -0,0 +1,186 @@
+from pytket.extensions.qiskit import AerBackend  # type: ignore
+from collections import Counter
+from pytket.backends.backendresult import BackendResult
+from pytket.utils.outcomearray import OutcomeArray
+import uuid
+from pytket.passes import BasePass
+from typing import Dict, List, Optional, Iterator
+from pytket import Circuit, Bit
+
+
+class TranspilerBackend:
+    """
+    Provides a backend like interface for noise simulation via compiler passes.
```

Comment:
Is there a reason not to make this an inherited class of `Backend`?

Reply:
I have vague recollections of that causing a problem, although I don't remember what they were. I may think about that again but for now I don't think there is anything I want to inherit from Backend here.